sycl/test-e2e/Basic/group_local_memory.cpp

```diff
@@ -7,17 +7,22 @@ constexpr int N = 5;
 
 int main() {
   sycl::queue q;
-  int *ptr = sycl::malloc_shared<int>(N, q);
-  q.parallel_for(sycl::nd_range<1>{1, 1}, [=](sycl::nd_item<1> it) {
-     auto g = it.get_group();
-     auto mem = sycl::ext::oneapi::group_local_memory<int[N]>(g, 1, 2, 3, 4, 5);
-     auto ref = *mem;
-     for (int i = 0; i < N; ++i) {
-       ptr[i] = ref[i];
-     }
-   }).wait();
+  sycl::buffer<int> buf{sycl::range{N}};
+  q.submit([&](sycl::handler &h) {
+    sycl::accessor acc{buf, h};
+    h.parallel_for(sycl::nd_range<1>{1, 1}, [=](sycl::nd_item<1> it) {
+      auto g = it.get_group();
+      auto mem =
+          sycl::ext::oneapi::group_local_memory<int[N]>(g, 1, 2, 3, 4, 5);
+      auto ref = *mem;
+      for (int i = 0; i < N; ++i) {
+        acc[i] = ref[i];
+      }
+    });
+  });
+  sycl::host_accessor result{buf};
   for (int i = 0; i < N; ++i) {
-    assert(ptr[i] == (i + 1));
+    std::cout << result[i] << std::endl;
```

Comment:
```suggestion
```

No need for any output in a positive case. It is not visible in logs and only increases I/O usage of a test for now reason.
Output for negative cases can be left in tests, because logs for failed tests are displayed and such output is usually helpful during initial analysis and debugging

Reply:
Good catch, I used this for debugging and then forgot it there apparently!